<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-6H0FSJA9" failures="1" tests="2" name="Error_validations" time="47.024" errors="0" timestamp="2024-07-26T15:39:31 IST">
  <testcase classname="Practice.Error_validations" name="login_test" time="1.751"/>
  <testcase classname="Practice.Error_validations" name="product_error_validations" time="23.849">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Practice.Error_validations.product_error_validations(Error_validations.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- product_error_validations -->
</testsuite> <!-- Error_validations -->
